devrunscript.sh

```
@@ -0,0 +1 @@
+./ResetDB.sh && docker compose up -d && cd API-REST/ && ./create-all-users.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)